src/logging.gleam

```diff
@@ -33,3 +33,17 @@ pub fn log(level: LogLevel, message: String) -> Nil {
 
 @external(erlang, "logger", "log")
 fn erlang_log(level: LogLevel, message: String) -> DoNotLeak
+
+@external(erlang, "logger", "set_primary_config")
+fn set_primary_config_level(key: Key, level: LogLevel) -> Nil
```

Comment:
```suggestion
fn set_primary_config_level(key: Key, level: LogLevel) -> DoNotLeak
```

Reply:
I'm curious on this bit, is this just a type purely set for developer documentation purposes? not doing any special stuff in the ffi or anything?

Reply:
It's a private type defined above to ensure that any unknown value doesn't leak in to the public interface. The previous typing was incorrect and due to being public could leak due to a programmer mistake, which could potentially cause runtime issues.